cache/class-vary-cache.php

```diff
@@ -348,13 +344,28 @@ private static function set_cookie( $name, $value ) {
 	}
 
 	/**
-	 * Wrapper for the set cookie function to slear out the cookie
+	 * Wrapper for the set cookie function to clear out the cookie
 	 *
 	 * @param string $name  Cookie Name.
 	 */
 	private static function unset_cookie( $name ) {
 		setcookie( $name, '', time() - 3600 );
 	}
+	/**
+	 * Only allow alphanumerics, dash and underscore
+	 *
+	 * @param string $value The string you want to test on.
+	 * @return WP_Error|boolean
+	 */
+	private static function validate_cookie_values( $value ) {
```

Comment:
Minor: This would be better named as a singular: `validate_cookie_value` (no `s`)